### ISSUE TITLE
Erroring on accents in source URLs

### ISSUE THREAD

**Issue by gi11es** [thumbor/thumbor#606]
```
curl -i http://127.0.0.1:8889/unsafe/100x100/127.0.0.1:8080/images/5/53/F%C3%A9lix.jpg
```

```
2015-11-11 22:33:35 thumbor:ERROR ERROR: Traceback (most recent call last):
File "/usr/local/lib/python2.7/site-packages/thumbor/handlers/init.py", line 102, in get_image
self.context.request.image_url
File "/usr/local/lib/python2.7/site-packages/tornado/gen.py", line 1008, in run
value = future.result()
File "/usr/local/lib/python2.7/site-packages/tornado/concurrent.py", line 232, in result
raise_exc_info(self.exc_info)
File "/usr/local/lib/python2.7/site-packages/tornado/gen.py", line 1017, in run
yielded = self.gen.send(value)
File "/usr/local/lib/python2.7/site-packages/thumbor/handlers/__init_.py", line 451, in _fetch
loader_result = yield self.context.modules.loader.load(self.context, url)
File "/usr/local/lib/python2.7/site-packages/tornado/concurrent.py", line 473, in wrapper
future.result()
File "/usr/local/lib/python2.7/site-packages/tornado/concurrent.py", line 232, in result
raise_exc_info(self._exc_info)
File "/usr/local/lib/python2.7/site-packages/tornado/concurrent.py", line 461, in wrapper
result = f(args, *kwargs)
File "/usr/local/lib/python2.7/site-packages/thumbor/loaders/http_loader.py", line 76, in load
load_sync(context, url, callback, normalize_url_func)
File "/usr/local/lib/python2.7/site-packages/thumbor/loaders/http_loader.py", line 94, in load_sync
url=encode(url),
File "/usr/local/lib/python2.7/site-packages/thumbor/loaders/http_loader.py", line 113, in encode
return None if string is None else string.encode('ascii')
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 35: ordinal not in range(128)

2015-11-11 22:33:35 thumbor:ERROR [BaseHandler] get_image failed for url 127.0.0.1%3A8080/images/5/53/F%C3%A9lix.jpg. error: 'ascii' codec can't decode byte 0xc3 in position 35: ordinal not in range(128)
2015-11-11 22:33:35 tornado.access:ERROR 500 GET /unsafe/100x100/127.0.0.1:8080/images/5/53/F%C3%A9lix.jpg (127.0.0.1) 3.33ms
```

**Comment by gi11es** [comment]
Might be related to #602

**Comment by gi11es** [comment]
Fixed by d9ed09cf7477604e2ddc663830f18054fc4a4958 or 1feff2a733e0a863584b483e9261c4aaa1929b25

**Comment by guilhermef** [comment]
\o/
